keras/wrappers/scikit_learn.py

```diff
@@ -115,23 +151,17 @@ def fit(self, X, y):
         history : object
             Returns details about the training history at each epoch.
         """
-        if len(y.shape) == 1:
-            self.classes_ = list(np.unique(y))
-            if self.loss == 'categorical_crossentropy':
-                y = to_categorical(y)
-        else:
-            self.classes_ = np.arange(0, y.shape[1])
 
-        self.compiled_model_ = copy.deepcopy(self.model)
+        self.compiled_model_ = copy.deepcopy(self.model.model)
```

Comment:
I'll change this. The reason why the underscore was at the end is related to a sklearn convention: every public attribute that is set in the fit method has an undescore in the end. I guess one can argue that the compiled_model should be a private attribute instead, so I'll change the position of the underscore.